src/pages/lesson/lesson.tsx

```diff
@@ -166,8 +166,9 @@ const Lesson: FC = () => {
                   <Markdown>{lesson.description ?? ''}</Markdown>
                 )}
 
-                {/* TODO https://github.com/Studio-Yandex-Practicum/lizaalert_frontend/issues/415 */}
-                {lessonType === 'Videolesson' && <VideoLesson source="" />}
+                {lessonType === 'Videolesson' && lesson.video_link !== null && (
```

Comment:
```suggestion
                {lessonType === 'Videolesson' && lesson.video_link && (
```

Если придет пустая строка, то мы тоже ничего не покажем